presidio-anonymizer/presidio_anonymizer/anonymizers/hash.py

```diff
@@ -36,5 +36,9 @@ def validate(self, params: dict = None) -> None:
         )
         pass
 
+    def anonymizer_name(self) -> str:
```

Comment:
Could be a parameter on the abstract class:
```python
self.name = self.__class__.__name__.lower()
```

Reply:
See my comment above, we wanted to implicitly state the name, WDYT?

Reply:
I'm debating between this and a class method, this does seems more elegant :)

Reply:
meaning this as default implementation and have the ability to override in the implementing class?

Reply:
Yes, this way it is both configurable but also works out of the box